package.json

```diff
@@ -40,7 +41,8 @@
       "canvas": "./src/util/shim/canvas.js",
       "eventemitter2": "./src/util/shim/EventEmitter2.js",
       "ws": "./src/util/shim/WebSocket.js",
-      "xmldom": "./src/util/shim/xmldom.js"
+      "xmldom": "./src/util/shim/xmldom.js",
+      "../util/decompressPng": "./src/util/shim/decompressPng"
```

Comment:
That's what I also thought but this is the way aliasify works apparently :disappointed: We could rewrite it using the [browser field of the package.json](https://github.com/substack/browserify-handbook#browser-field) and not use aliasify.